pennylane/_qubit_device.py

```diff
@@ -1273,6 +1286,65 @@ def classical_shadow(self, wires, circuit, seed=None):
 
         return self._cast(self._stack([outcomes, recipes]), dtype=np.int8)
 
+    def shadow_expval(self, obs, circuit):
+        r"""Compute expectation values using classical shadows in a differentiable manner.
+
+        The canonical way of computing expectation values is to simply average the expectation values for each local snapshot, :math:`\langle O \rangle = \sum_t \text{tr}(\rho^{(t)}O) / T`.
+        This corresponds to the case ``k=1``. However, it is often desirable for better accuracy to split the ``T`` measurements into ``k`` equal parts to compute the median of means, see `2002.08953 <https://arxiv.org/abs/2002.08953>`_.
+
+        One of the main perks of classical shadows is being able to compute many different expectation values by classically post-processing the same measurements. This is helpful in general as it may help
+        save quantum circuit executions.
+
+        Args:
+            H (:class:`~.pennylane.Hamiltonian` or :class:`~.pennylane.operation.Tensor`): Observable to compute the expectation value over.
+            k (int): Number of equal parts to split the shadow's measurements to compute the median of means. ``k=1`` corresponds to simply taking the mean over all measurements.
+
+        Returns:
+            float: expectation value estimate.
+
+        .. note::
+
+            This measurement uses the measurement :func:`~.pennylane.classical_shadow` and :class:`~.pennylane.ClassicalShadow` for post-processing
+            internally to compute expectation values. In order to compute correct gradients using PennyLane's automatic differentiation,
+            you need to use this measurement.
+
+        **Example**
+
+        .. code-block:: python3
+
+            H = qml.Hamiltonian([1., 1.], [qml.PauliZ(0)@qml.PauliZ(1), qml.PauliX(0)@qml.PauliX(1)])
```

Comment:
```suggestion
            H = qml.Hamiltonian([1., 1.], [qml.PauliZ(0) @ qml.PauliZ(1), qml.PauliX(0) @ qml.PauliX(1)])
```

to ensure Sphinx doesn't render it using decorator syntax:

![image](https://user-images.githubusercontent.com/2959003/185969607-eaf8928d-f270-475c-abad-165ac1e1d29f.png)